cmd/update_test.go

```diff
@@ -141,4 +141,46 @@ var _ = Describe("Update", func() {
 			Eventually(session).Should(gexec.Exit(0))
 		})
 	})
+
+	Describe("When Github returns a 403 error", func() {
+		BeforeEach(func() {
+			command = exec.Command(pathCLI,
+				"update", "check",
+				"--github-api", tempSettings.TestServer.URL(),
```

Comment:
Just to make sure I understand this correctly - `--github-api` is a hidden flag to point the CLI to a test, fake instance of "GitHub API"?

Reply:
Correct, `--github-api` is a global flag which maps to the `GitHubAPI` field of the rootOptions:
https://github.com/CircleCI-Public/circleci-cli/blob/5896baa95dad1b66f9c4a5b0a14571717c92aa55/cmd/root.go#L79

The flag is created here, but hidden from the user:
https://github.com/CircleCI-Public/circleci-cli/blob/5896baa95dad1b66f9c4a5b0a14571717c92aa55/cmd/root.go#L137